src/trace_processor/importers/proto/track_event_tokenizer.cc

```diff
@@ -197,11 +198,22 @@ ModuleResult TrackEventTokenizer::TokenizeTrackDescriptorPacket(
     }
```

Comment:
There's a TokenizeThreadDescriptor() call right before, which eventually stores the tids into the sequence state (and they are read from there by parsing code later). I am not sure if keeping these inconsistent with the reservation etc works?

This all feels a bit more subtle than I thought ...